Sources/Guaka/Flag/Flag.swift

```diff
@@ -56,14 +56,31 @@ public struct Flag: Hashable {
   public let type: FlagValue.Type
 
 
-  /// Set the flag to be required/
+  /// Set the flag to be required
   /// A required flag must be set in order for commands to be executed.
   public let required: Bool
 
 
+  /// Flag values.
+  /// The value can be `int`, `bool`, `string` or any other type that implements `FlagValue`
+  public var values: [FlagValue] = []
```

Comment:
Change the underlying storage so that we can support both single and repeated flags